operations/deployment/terraform/elb.tf

```diff
@@ -0,0 +1,76 @@
+data "aws_acm_certificate" "issued" {
+  domain = var.domain_name
+}
+
+data "aws_elb_service_account" "main" {}
+resource "aws_s3_bucket" "lb_access_logs" {
+  bucket = var.lb_access_bucket_name
+
+  policy = <<POLICY
+{
+  "Id": "Policy",
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "s3:PutObject"
+      ],
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::${var.lb_access_bucket_name}/*",
+      "Principal": {
+        "AWS": [
+          "${data.aws_elb_service_account.main.arn}"
+        ]
+      }
+    }
+  ]
+}
+POLICY
+
+  tags = {
+    Name  = var.lb_access_bucket_name
+  }
+}
+
+resource "aws_s3_bucket_acl" "lb_access_logs_acl" {
+  bucket = aws_s3_bucket.lb_access_logs.id
+  acl    = "private"
+}
+
+resource "aws_elb" "vm" {
+  count              = "${var.sub_domain_name}.${var.domain_name}"  == null ? 0 : 1
+  name               = "${var.aws_resource_identifier}"
+  security_groups    = [aws_security_group.ec2_security_group.id]
+  availability_zones = ["us-east-1a", "us-east-1b", "us-east-1c", "us-east-1d", "us-east-1e", "us-east-1f"]
```

Comment:
should these match the `AWS_DEFAULT_REGION` param?

Reply:
Fixing in next commit